APIs/schemas/sender_transport_params.json

```diff
@@ -25,6 +26,23 @@
       "items": {
         "$ref": "sender_transport_params_mqtt.json"
       }
+    },
+    {
+      "type": "array",
+      "items": {
+        "description": "Any other definition for transports registered in the parameter register",
+        "type": "object",
+        "patternProperties": {
+          "^[a-zA-Z0-9_]+$": {
+            "type": [
+              "string",
+              "boolean",
+              "null",
+              "number"
+            ]
```

Comment:
Since it's a very simple set of requirements I would personally say it's better to directly mention them rather than refer to the _ext schema. Ext parameters have certain connotations which are defined in a specific markdown section [externally-defined-parameters](https://specs.amwa.tv/is-05/releases/v1.1.2/docs/Behaviour.html#externally-defined-parameters).